mkdocs.yml

```
@@ -16,6 +16,8 @@ nav:
     - Setup Slack Events API receiver: getting_started/setup_slack.md
```

```suggestion
    - Set up Slack Events API receiver: getting_started/setup_slack.md
```

(and all other imperative 'Setup's should be 'Set up's)